presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -437,7 +443,7 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
                 hiveStorageFormat,
                 partitionedBy,
                 bucketProperty,
-                additionalTableParameters,
+                additionalTableParameters.build(),
```

Comment:
Let's add `Optional<String> comment` as a parameter to this method, so the table property logic only happens in one place